src/agri_ai/langchain_tools/field_registration_tool.py

```
@@ -0,0 +1,252 @@
+        # 既存の同じプレフィックスの最大番号を取得
+        existing_fields = await collection.find(
+            {"field_code": {"$regex": f"^{prefix}-"}}
+        ).to_list(1000)
+        
+        existing_numbers = []
+        for field in existing_fields:
+            code = field.get("field_code", "")
+            try:
+                num = int(code.split("-")[1])
+                existing_numbers.append(num)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)